plugins/modules/ios_bfd_global.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2025 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+The module file for ios_bfd_global
+"""
+
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: ios_bfd_global
+extends_documentation_fragment:
+  - cisco.ios.ios
+short_description: Bidirectional Forwarding Detection (BFD) global-level configuration
+description:
+   - Manages Bidirectional Forwarding Detection (BFD) global-level configuration.
+version_added: 1.0.0
+author:
+  - Komal Desai (@komaldesai13)
+notes:
+  - Tested against Cisco IOS XE Software, Version 17.13.01a on CML
+  - This module works with connection C(network_cli)
+options:
+  slow_timer:
+    description:
+      - Configures slow timers used by BFD , Value in ms to use for slow timers  <1000-30000>
+    required: false
+    type: int
+  l2cos:
+    description:
+      - Configures L2 COS value to be used for BFD packets over VLAN interfaces <0-7>
+    required: false
+    type: int
+  bfd_template:
+    description: creates a reusable template that defines a set of Bidirectional Forwarding Detection (BFD) session parameters
+    required: false
+    type: dict
+    suboptions:
+      name:
+        description: name of the BFD template to be used
+        required: true
+        type: str
+      hop:
+        description: type of template to be used
+        required: true
+        choices:
+          - single-hop
+          - multi-hop
+        type: str
+      config:
```

Comment:
Config here is contradictory, we have to look for an alternate name for this attribute 
A best practice is to keep config at the top always